cantabular/table.go

```diff
@@ -298,11 +298,16 @@ func decodeValues(ctx context.Context, dec jsonstream.Decoder, dims Dimensions,
 // createCSVHeader creates an array of strings corresponding to a csv header
 // where each column contains the value of the corresponding dimension, with the last column being the 'count'
 func createCSVHeader(dims Dimensions) []string {
-	header := make([]string, len(dims)+1)
-	for i, dim := range dims {
-		header[i+1] = dim.Variable.Label
+	i := 0
+	l := len(dims) * 2
+	header := make([]string, l+1)
+	for _, dim := range dims {
```

Comment:
Could you initialise `i` here in the loop declaration and remove the second incrementation?

Reply:
I am not sure this would be a better option

Reply:
I would still stick with my initial approach, as it is easier to read the code.

Reply:
I am approving this as either approach are acceptable and won't effect the outcome